src/akamai/lib/AkamaiHDPlayer.as

```diff
@@ -246,12 +246,15 @@ package
       var caption : Object = info["parameters"];
       for (var key:String in caption)
       {
-        var captionText : String = caption[key];
-        if (captionText != null && _mode !="disabled")
+        if(key.substr(0,4)=="data")
```

Comment:
Do all captions coming through this method start with "data" then? What was happening before this fix?